packages/react-debug-tools/src/ReactDebugTools.js

```diff
@@ -8,5 +8,6 @@
  */
 
 import {inspectHooks, inspectHooksOfFiber} from './ReactDebugHooks';
+import {ErrorsNames} from './ReactDebugCustomErrors';
 
-export {inspectHooks, inspectHooksOfFiber};
+export {inspectHooks, inspectHooksOfFiber, ErrorsNames};
```

Comment:
Exporting error constants as part of the `ReactDebugTools` exports seems a little heavy handed, and they don't feel like they should be a part of the `react-debug-tools` API. What do you think about just checking for the error string inside devtools instead? This also feels a bit janky but we have precedent of doing this already, and we can later refactor the code to use a custom error if/when we move this codebase into devtools

Reply:
If we are not adding many strings in foreseeable future (which should be true IMO) I'm OK with that.